bundle/src/main/groovy/be/orbinson/aem/groovy/console/servlets/ScriptPostServlet.groovy

```diff
@@ -74,16 +69,16 @@ class ScriptPostServlet extends AbstractJsonResponseServlet {
                 request: request,
                 response: response,
                 outputStream: outputStream,
-                printStream: new PrintStream(outputStream, true, Charsets.UTF_8.name()),
-                script: checkNotNull(getScript(request, scriptPath), "Script cannot be empty.")
+                printStream: new PrintStream(outputStream, true, "UTF-8"),
```

Comment:
You can use https://docs.oracle.com/javase/8/docs/api/java/nio/charset/StandardCharsets.html#UTF_8 with its `name()` method

Reply:
Java 10 also added https://docs.oracle.com/en%2Fjava%2Fjavase%2F21%2Fdocs%2Fapi%2F%2F/java.base/java/io/PrintStream.html#%3Cinit%3E(java.io.File,java.nio.charset.Charset).